packages/webviz-core/src/panels/Rosout/index.stories.js

```diff
@@ -93,6 +93,17 @@ storiesOf("<RosoutPanel>", module)
       </PanelSetup>
     );
   })
+  .add("topicToRender", () => {
+    return (
+      <PanelSetup
+        fixture={{
+          topics: [{ name: "/foo/rosout", datatype: "dummy" }],
+          frame: { "/foo/rosout": fixture.frame["/rosout"].map((msg) => ({ ...msg, topic: "/foo/rosout" })) },
```

Comment:
Could be nice to also test with a prefixed topic like `/webviz_bag_2/some_topic_with_correct_datatype`.